pubsub/README.rst

```diff
@@ -158,3 +158,35 @@ block the current thread until a given condition obtains:
 To learn more, consult the `subscriber documentation`_.
 
 .. _subscriber documentation: https://googleapis.github.io/google-cloud-python/latest/pubsub/subscriber/index.html
+
+
+Authentication
+^^^^^^^^^^^^^^
+
+It is possible to specify the authentication method to use with the Pub/Sub
+clients. This can be done by providing an explicit `Credentials`_ instance from
```

Comment:
It would be nice to add a bit more emphasis on the google auth link, e.g.: 

```
This can be done by providing an explicit `Credentials`_ instance. Support for various authentication methods is available from the `google-auth` library.
```

Reply:
Sure thing, will add right away. Thanks for the feedback!